docs/reference/logging/fluentd.md

```diff
@@ -0,0 +1,86 @@
+# Fluentd logging driver
+
+The `fluentd` logging driver sends container logs to the [Fluentd](http://www.fluentd.org/)
+as structured log data. Users can write these logs into any file
```

Comment:
"Users can write these logs into any file systems or storages by using many output plugins."

How about
Then users can utilize any of the [various output plugins of Fluentd](http://www.fluentd.org/plugins) in order to write these logs to variety of destinations.
?